channeldb/invoice_test.go

```diff
@@ -228,6 +280,44 @@ func TestInvoiceWorkflow(t *testing.T) {
 	}
 }
 
+// TestPayHashIndexFallback asserts that the invoice db properly prioritizes
+// lookups and updtes by payment address.
+func TestPayHashIndexFallback(t *testing.T) {
+	db, cleanUp, err := makeTestDB()
+	defer cleanUp()
+	assert.Nil(t, err)
```

Comment:
Nice those assertions. Do we have enough approvals from the team to move to `testify` for `lnd`?